### PR TITLE
Add Quake 2 RTX test profile

### PR DESCRIPTION
**Pull request by Wozard**
Hello!

We have developed a test profile for Quake II RTX, which we would like to contribute to the Phoronix Test Suite. Let me know if you have any questions, comments, or concerns about the profile!

Cheers,
Gary Dunn
NVIDIA
gdunn@nvidia.com

### CODE REVIEW

File: pts/quake2rtx-1.0.0/test-definition.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+The test profile assumes you have a Steam account, and have Steam installed for the system. This automates the process of executing the game and using its built-in benchmark mode.
+    </Description>
+    <ResultScale>Frames Per Second</ResultScale>
+    <Proportion>HIB</Proportion>
+    <Executable>q2rtx.sh</Executable>
+    <TimesToRun>3</TimesToRun>
+  </TestInformation>
+  <TestProfile>
+    <Version>1.0</Version>
```
Comment (michaellarabel) [inline pts/quake2rtx-1.0.0/test-definition.xml:17]: For future reference, it should be 1.0.0. x.y.z format is mandated during the schema validation. Will correct it when finishing reviewing this test profile and uploading it to OpenBenchmarking.org.

File: pts/quake2rtx-1.0.0/pre.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+# Input settings
+WIDTH=$1
+HEIGHT=$2
+
+# Set resolution
+echo -n "$( xdpyinfo | grep -oP 'dimensions:\s+\K\S+' )" > /tmp/old_resolution
```
Comment (michaellarabel) [inline pts/quake2rtx-1.0.0/pre.sh:8]: This way works fine though for reference there is $VIDEO_WIDTH and $VIDEO_HEIGHT exported to test profiles (a list of available env vars can be found via `phoronix-test-suite diagnostics`) to also get the original resolution as of start time (with various fallbacks/safeguards for querying the resolution via various means). I think most distributions though are still shipping with xdpyinfo by default so this way works fine as well sans should any other test being run have failed to properly restore the mode during test that this test would then be querying the non-optimal resolution.

Comment (Wozard) [reply pts/quake2rtx-1.0.0/pre.sh:8]: That's good to know! I'll switch over to using those vars

**Comment by michaellarabel** [comment]
Thanks, seems to be working well. Now merged in through OpenBenchmarking.org. Will be running some tests on some more graphics cards.

https://openbenchmarking.org/test/pts/quake2rtx-1.0.0

**Comment by Wozard** [comment]
Hello again!

I was testing the live test profile for Quake 2 RTX and noticed that resolution had no impact on performance.

It looks like the `pre.sh` script was accidentally omitted from the test profile while pushing to live: [https://github.com/phoronix-test-suite/test-profiles/tree/master/pts/quake2rtx-1.0.0](https://github.com/phoronix-test-suite/test-profiles/tree/master/pts/quake2rtx-1.0.0)

Would you mind resubmitting the profile with the `pre.sh` script? I can submit another pull request if that makes thing easier for you!

Cheers,
Gary Dunn
NVIDIA
gdunn@nvidia.com

**Comment by michaellarabel** [comment]
Ah good catch, something got messed up when originally cloning it and reviewing it locally. In any case pts/quake2rtx-1.0.1 is now pushed out that restores the pre.sh. Thanks.

**Pull request closed without merging**